modules/views/civicrm/civicrm_handler_filter_state_multi.inc

```diff
@@ -89,63 +88,65 @@ class civicrm_handler_filter_state_multi extends views_handler_filter_in_operato
     if (!empty($country_field)) {
       $this->value_options = $loaded_list;
 
-      $fid=md5($country_field);
-      $js = '(function ($) {
-               Drupal.behaviors.civicrm_views_' . $fid . ' = {
+      $fid = md5($country_field);
+      $js = "(function ($) {
+               Drupal.behaviors.civicrm_views_{$fid}= {
                  attach: function (context, settings) {
-                   $("#edit-' . $country_field . '").change( function() {
-                     var stateList = \'<option value="All">&lt;Any&gt;</option>\';
+                   $(#" . '"' . "edit-{$country_field}" . '"' . ").change( function() {
```

Comment:
This is not the same. Note the hash mark is now outside the quotes.

Reply:
Thanks @colemanw